thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -140,21 +140,46 @@ class CDistributorBase : implements IHashDistributor, implements IExceptionHandl
              */
             dedupList.sort(*iCompare, 1);
```

Comment:
No this is not a stable sort, because the send/recv order involve here is indeterminate so no point.
Should it be for BEST though?